catalog/index.js

```diff
@@ -21,6 +21,7 @@ import modal from "./pages/modal";
 import calendarView from "./pages/calendar_view";
 import popover from "./pages/popover";
 import tooltip from "./pages/tooltip";
+import seattooltip from "./pages/seattooltip";
```

Comment:
There's no need to create a new catalog page for this. It can go on the tooltip page as well